### PR TITLE
Update search function and css for cards

### PR DESCRIPTION
**Pull request by rdren0**

### CODE REVIEW

**Review by rdren0 — COMMENTED** [review]
Select Color Function works great and is DRY.
Update to props now that we have urls instead of stored locally is accurate
Will need to start working more on testing

File: package.json [hunk 1]
```diff
@@ -21,5 +22,15 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
```
Comment (Klhalls89) [inline package.json:25]: Thanks for setting up our first snapshot test, it looks great.

File: src/components/GemCardStyle.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+*{
+  box-sizing: border-box;
```
Comment (Klhalls89) [inline src/components/GemCardStyle.scss:2]: I'm glad we have enough styling now to see our results clearly

**Comment by IsaacSunoo** [comment]
Getting gems by the color and name look good! Nice work using a single state to store either of those filters into. We have to work on doing the same with the metals now.

**Pull request merged**